app/services/spotlight/carrierwave_file_resolver.rb

```diff
@@ -9,10 +9,10 @@ def initialize
     end
 
     def pattern(id)
-      uploaded_file = Spotlight::FeaturedImage.find(id).image.file
+      uploaded_file = Spotlight::FeaturedImage.find(id).image.blob
       raise Riiif::ImageNotFoundError, "unable to find file for #{id}" if uploaded_file.nil?
 
-      uploaded_file.file
+      ActiveStorage::Blob.service.path_for(uploaded_file.key)
```

Comment:
I'm not sure how this will work with other types of ActiveStorage::Service, but should those people just write their own file resolver?

Reply:
Yeah..  I think we might have to adapt RIIIF to handle some of this better. I think  ` ActiveStorage::Blob.service.path_for` won't work with some of the remote storage options, right?

(but... the whole download-with-a-block API won't play nice with RIIIF)